torch/csrc/jit/codegen/cuda/compute_at_map.cpp

```diff
@@ -801,101 +834,6 @@ bool ComputeAtMap::areMapped(
   return disjointSetOf(id0, mode)->has(id1);
 }
 
-namespace {
-
-// Validate a LOOP concrete ID has the complete ID set required for
-// indexing. See issue #1655 and FusionIncompleteConcreteID for an
-// example fusion that fails with this validation. Fixing this issue
-// would require creating a reference IterDomain with all the
-// necessary root ID for for loop extent generation, for indexing, and for
-// predication.
-//
-// root_ids_of_all_ids and root_ids_of_concrete_id consist of EXACT
-// concrete IDs.
-void validateCompletenessOfLoopConcreteID(
```

Comment:
Still trying to understand the changes of this PR. Why is this check no longer necessary?

Reply:
Because if it's not resolved than a concrete id wouldn't be found, so instead of erroring in validation it just won't be able to find the concrete id.